File: js/escritura.js
const yaml = require('js-yaml');
const fs   = require('fs');

const data = {
  nombre: 'Juan',
  edad: 30,
  intereses: ['JavaScript', 'lectura', 'deportes']
};

// Escribe el objeto JavaScript como YAML en un archivo
try {
  fs.writeFileSync('../archyml/output.yaml', yaml.dump(data), 'utf8');
} catch (e) {
  console.log(e);
}

File: python/escritura.py
import yaml

data = {
    'nombre': 'Juan',
    'edad': 30,
    'intereses': ['python', 'lectura', 'deportes']
}

# Escribe el diccionario como YAML en un archivo
with open('../archyml/output.yaml', 'w') as file:
    yaml.dump(data, file)
